www/_posts/2017-09-25-ios-release.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+author:
+    name: Suraj Pindoria
+    url: https://twitter.com/surajpindoria
+title:  "Cordova iOS 4.5.1"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce a minor version of `Cordova iOS 4.5.1` has been released!
+
+This version provides updates for the latest iOS 11 and Xcode 9. You can now create builds for this new version and properly deploy to either the emulator or device.
+
+Things to note:
+
+* iPhone X new screen size, must use Launch Storyboards to take full advantage
```

Comment:
grammar: For iPhone X's new screen size, you must use Launch Storyboards to take full advantage of it